framework/src/st_things/st_things.c

```diff
@@ -277,32 +277,26 @@ int st_things_deinitialize(void)
 {
 	THINGS_LOG_D(TAG, THINGS_FUNC_ENTRY);
 
-	if (STACK_INITIALIZED != g_stack_status) {
-		int ret_val = ST_THINGS_ERROR_OPERATION_FAILED;
-		switch (g_stack_status) {
-		case STACK_NOT_INITIALIZED:
-			THINGS_LOG_E(TAG, "Stack is not initialized.");
-			ret_val = ST_THINGS_ERROR_STACK_NOT_INITIALIZED;
-			break;
-		case STACK_STARTED:
-			THINGS_LOG_E(TAG, "Stack is currently running. Stop the stack before deinitializing it.");
-			ret_val = ST_THINGS_ERROR_STACK_RUNNING;
-			break;
-		default:
-			THINGS_LOG_E(TAG, "Invalid stack state: %d.", g_stack_status);
-			break;
-		}
+	if (STACK_INITIALIZED == g_stack_status) {
+		THINGS_LOG_E(TAG, "Stack is initialized.");
```

Comment:
this is not ERROR.
How about you change to LOG_V?

Reply:
Fixed.